### DISCUSSION TITLE
[BUG] unhandled exception

### DISCUSSION THREAD

**Discussion by spamkutu** [Chia-Network/chia-blockchain#8404] [Support]
**Describe the bug**
	Bucket 120 uniform sort. Ram: 3.739GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Bucket 121 uniform sort. Ram: 3.739GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
	Bucket 122 uniform sort. Ram: 3.739GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
	Bucket 123 uniform sort. Ram: 3.739GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
	Bucket 124 uniform sort. Ram: 3.739GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Bucket 125 uniform sort. Ram: 3.739GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
	Bucket 126 uniform sort. Ram: 3.739GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Bucket 127 uniform sort. Ram: 3.739GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Total matches: 4294855134
Caught plotting error: Matches do not match with number of write entries 4294855134 4294854857
Traceback (most recent call last):
  File "D:\a\chia-blockchain\chia-blockchain\venv\Lib\site-packages\chia\cmds\chia.py", line 132, in <module>
  File "D:\a\chia-blockchain\chia-blockchain\venv\Lib\site-packages\chia\cmds\chia.py", line 128, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 153, in create_cmd
  File "asyncio\base_events.py", line 642, in run_until_complete
  File "chia\plotting\create_plots.py", line 247, in create_plots
RuntimeError: Unknown exception
[21328] Failed to execute script 'chia' due to unhandled exception!

**To Reproduce**

Time to time its self

**Desktop**

- OS: Windows 10
- OS Version/Flavor:  Pro
- CPU: Intel(R) Core(TM) i7-10710U CPU

**Comment by dreamboatcap** [comment]
https://chiaforum.com/t/1-1-6-plotting-hangs-with-matches-do-not-match-with-number-of-write-entries-error/4789/3

**Comment by spamkutu** [comment]
Thank you i will follow there

**Comment by loppefaaret** [comment]
This mostly down to hardware - likely memory, try and cool them better, turn off XMP or overclocking, and see if you get through plotting

**Comment by loppefaaret** [comment]
any luck ?

**Comment by spamkutu** [comment]
no luck after deleted my config.yaml and restarted the system its now see the pilot and its look good like working to produce but all my pilot was nft pilot and this time there is no pool i made alot nft but my pilot belong to nfts. so this time its not show me the my contracted nft pools its always say join a pool, i dont need to join any more nfts pool atm. i just want to recall my old conractted nfts pool how that will be possible? I am totaly stuck now on this hardvester its show my pilot all green and working produce state but there no any pool or nfts how that is possible? And also i cannot produce my new pilots since i dont see my nfts contracted pool so i am realy stuck atm, help me to recall my conracted pools to back please,
many many thanks loppefaaret,
i not follow this since there told me to goo ill check there so sorry to my latency answer and waiting your new clue thank you so so much to you!!
https://chiaforum.com/t/1-1-6-plotting-hangs-with-matches-do-not-match-with-number-of-write-entries-error/4789/3

**Reply by Evol3000** [reply]
The same happened to me, What I did was to format the temporary storages, in my case 2 nvme. I did a quick format to both of them and that solved the problem.

**Comment by BrandtH22** [comment]
@spamkutu , the newest version of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```
NOTE - when recreating the config.yaml you can perform a full resync of the wallet (stop chia, delete the wallet db, start chia) to autopopulate the pool information.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
